ush/rocoto/setup_workflow.py

```diff
@@ -197,6 +197,8 @@ def get_definitions(base):
     if scheduler in ['slurm']:
         strings.append(f'''\t<!ENTITY PARTITION_SERVICE "{base['QUEUE_SERVICE']}">\n''')
     strings.append(f'\t<!ENTITY SCHEDULER  "{scheduler}">\n')
+    if machine in ['WCOSS2']:
+       strings.append(f'\t<!ENTITY NODESIZE  "128">\n')
```

Comment:
Should this be based on the nodesize set above?

Reply:
It should...this is just hardcoded for ease/time. See comment below in PR.